### DISCUSSION TITLE
Color input, color picker

### DISCUSSION THREAD

**Discussion by ibrahimtuksal** [cosscom/coss#181] [Suggestions]
for example:

![image](https://github.com/user-attachments/assets/26dccd2b-2879-43d7-8c32-d1a41975d29a)
![image](https://github.com/user-attachments/assets/1448b969-0608-4470-8a5d-25a3eac08bf4)

**Comment by Ayanavo** [comment]
It will be great to have a color input field, but versatile enough to accomodate a color field, color button, and a color picker pop up event handler to open it under any condition anywhere.

**Comment by jorgedanisc** [comment]
The absolute best color input I found until now is from [Adobe Spectrum](https://react-spectrum.adobe.com/react-spectrum/ColorPicker.html). However their biggest problem is the ease of use, so I can see an opportunity to make something amazing from it!

<img width="185" alt="Screenshot 2025-04-09 at 15 42 41" src="https://github.com/user-attachments/assets/6bd81de7-aedc-4b5c-9a49-150be2b305ac" />
<img width="297" alt="Screenshot 2025-04-09 at 15 42 26" src="https://github.com/user-attachments/assets/214cd07e-912b-4803-be8a-02d1883f51ef" />

**Comment by karticme** [comment]
Here is one from my side which you can use [Color Picker](https://bykartic.vercel.app/color-picker)
All suggestions are welcome.

> I love to discuss more about it if you intrested @pasqualevitiello and If it will be in @shadcn then 😎 for me.
